docs/node-operators/how-to/use-snapshots.md

```diff
@@ -10,133 +10,114 @@ import TabItem from '@theme/TabItem';
 
-You can now remove all previous states of the chain by running the reset commands, either together or separately.
+## Restarting a node using local snapshots
 
-:::caution Reset clears all data
-Ensure that your Vega and Tendermint config homes are different, as this command will delete everything within the home folder. If, for example, you reset Tendermint and everything is in the same folder or choose the Vega home path rather than Tendermint, it will also delete your Vega config and saved keys.
-:::
+The default action when starting a node is for it to load from the latest snapshot that exists locally. This means if you have stopped your node for any reason, and simply want to catch up with the network then there is nothing to do.
 
+As an example, if the local snapshot list looks like the below:
+```shell
+Snapshots available: 2
+	Height 901, version: 4, size 92, hash: 562414bb5be3ccc8403fbd030d06eebc799bfef5ca25b02ad360fec349cb4bc8
+	Height 601, version: 3, size 92, hash: 72a2edd960cf3340ae94bf092991f923850738144789959124590675798fefd9
+```
 
+then starting a node as follows will mean that it will start processing from block height `901`
 ```
-vega tendermint unsafe_reset_all --home="YOUR_TENDERMINT_HOME"
-vega unsafe_reset_all --home="YOUR_VEGA_HOME"
+vega start --home="YOUR_VEGA_HOME" --tendermint-home="YOUR_TENDERMINT_HOME"
 ```
 
-## Inspect snapshots data using vegatools
- 
-:::note
-This is not required to deploy / use the snapshots. This section shows how to use `vega tools` to inspect the data stored in the snapshot.
-:::
-
-Vega tools offer basic utilities to read the data from a snapshots and dump it into json format.
-
-If you have `go` and `git` configured on your environment you can install vegatools like so:
+To start a node from a more historic snapshot the CLI option `--snapshot.load-from-block-height=` can be used. For example the following will start the node from a block height of `601`
 ```
-git clone git@github.com:vegaprotocol/vegatools.git && cd vegatools && go install
+vega start --snapshot.load-from-block-height=601 --home="YOUR_VEGA_HOME" --tendermint-home="YOUR_TENDERMINT_HOME"
 ```
 
-Then using the vegatools command line you can inspect the state of the snapshot at a given height:
+
+## Restarting a node using a network snapshot
+
+:::caution Using network snapshots if local snapshots exist should be avoided
+Restarting a node from a network snapshot requires the removal of all existing Tendermint block-data and Vega state data. This should only be done if absolutely necessary. Replaying from a local snapshot is always preferred.
+:::
+
+Restarting an existing node from a network snapshot requires removal of all existing local node data. This can be done by running the below commands and then following the above section for starting a new node using network snapshots. Note that there are very few reasons where a node operator will need to do this
```

Comment:
```suggestion
Restarting an existing node from a network snapshot requires removal of all existing local node data. This can be done by running the below commands and then following the above section for starting a new node using network snapshots. Note that there are very few reasons that a node operator will need to do this.

```